src/emoji/realmEmojiReducers.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import { StateType, Action } from '../types';
+import {
+  EVENT_REALM_EMOJI_UPDATE,
+  LOGOUT,
+  LOGIN_SUCCESS,
+  ACCOUNT_SWITCH,
+  INIT_REALM_EMOJI,
+} from '../actionConstants';
+
+const initialState = {
+  realm_emoji: {},
```

Comment:
Rename to realmEmoji, we try to keep realm_emoji casing only for API entities. Even better, when in the realm reducer, just rename to `emoji`.